nagios/datadog_checks/nagios/nagios.py

```diff
@@ -331,8 +331,31 @@ def _parse_line(self, line):
     def create_event(self, timestamp, event_type, hostname, fields):
         """Factory method called by the parsers
         """
+        # The Agent expects a specific set of fields, so we need to place all
```

Comment:
Where does this specific set come from?

Reply:
It comes from
Agent 5 - https://github.com/DataDog/dd-agent/blob/0137a836d7ac3390c6d298edbc082567bd306437/aggregator.py#L654
vs
Agent 6 - https://github.com/DataDog/datadog-agent/blob/c0d3dc9bdd3ce0dbc379c01b010487e927464095/pkg/metrics/event.go#L72

Agent 5 allows for a more dynamic setup where you can send more keys, but Agent 6 is a little stricter on this, so you can only send whats defined in the struct.